yarpl/include/yarpl/flowable/Subscriber.h

```diff
@@ -29,6 +29,10 @@ class Subscriber : public virtual Refcounted {
     subscription_.reset();
   }
 
+  void onError() {
```

Comment:
This doesn't seem necessary.  The only use is in `FramedWriter::onError(eptr)`, and you can just call `SubscriberBase::onError(eptr)`.  It incurs a useless copy of `eptr`, but I think that's better than adding this overload.